src/main/scala/com/atomist/rug/kind/dynamic/ContainerTreeNodeView.scala

```diff
@@ -12,7 +12,7 @@ class ContainerTreeNodeView[O <: ContainerTreeNode](
                                                      originalBackingObject: O,
                                                      parent: MutableView[_])
   extends ViewSupport[O](originalBackingObject, parent)
-    with FormatInfoProvider {
+    with FormatInfoProviderSupport {
```

Comment:
This renaming necessary to avoid generation being confused with methods with the same name from different traits